File: test/fixtures/output/csharp/httpclient/full.cs
var clientHandler = new HttpClientHandler
{
    UseCookies = false,
};
var client = new HttpClient(clientHandler);
var request = new HttpRequestMessage
{
    Method = HttpMethod.Post,
    RequestUri = new Uri("http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value"),
    Headers =
    {
        { "cookie", "foo=bar; bar=baz" },
        { "accept", "application/json" },
    },
    Content = new FormUrlEncodedContent(new Dictionary<string, string>
    {
        { "foo", "bar" },
    }),
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}

File: test/fixtures/output/java/asynchttp/full.java
AsyncHttpClient client = new DefaultAsyncHttpClient();
client.prepare("POST", "http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value")
  .setHeader("cookie", "foo=bar; bar=baz")
  .setHeader("accept", "application/json")
  .setHeader("content-type", "application/x-www-form-urlencoded")
  .setBody("foo=bar")
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

client.close();
